Tests/Fixtures/Generated/IB-iOS/segues-swift3-context-all-customname.swift

```diff
@@ -34,12 +34,19 @@ internal enum XCTStoryboardCustom {
 
 // MARK: - Implementation Details
 
-internal protocol SegueType: RawRepresentable { }
+internal protocol SegueType: RawRepresentable {}
 
 internal extension UIViewController {
   func perform<S: SegueType>(segue: S, sender: Any? = nil) where S.RawValue == String {
     performSegue(withIdentifier: segue.rawValue, sender: sender)
   }
 }
 
+internal extension SegueType {
+  init?(_ segue: UIStoryboardSegue) {
+    guard let id = segue.identifier else { return nil }
+    return self(rawValue: id)
```

Comment:
Did you check if this code compiled? I never remember if we can `return` anything other than `nil` from a `init?`, aren't we forced to do `self = self(rawValue: id)` instead?

(I guess our tests will catch that on CI if it doesn't compile…)